lib/lambda_ethereum_consensus/state_transition/accessors.ex

```diff
@@ -178,9 +179,10 @@ defmodule LambdaEthereumConsensus.StateTransition.Accessors do
   Return the randao mix at a recent ``epoch``.
   """
   @spec get_randao_mix(BeaconState.t(), Types.epoch()) :: Types.bytes32()
+  @spec get_randao_mix(BeaconState.t(), Types.epoch()) :: Types.bytes32()
   def get_randao_mix(%BeaconState{randao_mixes: randao_mixes}, epoch) do
-    epochs_per_historical_vector = ChainSpec.get("EPOCHS_PER_HISTORICAL_VECTOR")
-    Enum.fetch!(randao_mixes, rem(epoch, epochs_per_historical_vector))
+    index = Randao.get_randao_mix_index(epoch)
+    Aja.Vector.at!(randao_mixes, index)
```

Comment:
We should move this to a `Randao.get_randao_mix` function. Also, we can remove this function, and just call `Randao.get_randao_mix(state.randao_mixes, epoch)` where used